cdk/bin/cdk.ts

```diff
@@ -1,13 +1,29 @@
 import "source-map-support/register";
 import { App } from "aws-cdk-lib";
+import type { MobileSaveForLaterProps } from "../lib/mobile-save-for-later";
 import { MobileSaveForLater } from "../lib/mobile-save-for-later";
 
 const app = new App();
-new MobileSaveForLater(app, "MobileSaveForLater-CODE", {
+
+export const codeProps: MobileSaveForLaterProps = {
```

Comment:
This is the main advantage of using two separate templates (see https://github.com/guardian/mobile-save-for-later/commit/ad96d92d7c997a404d86f9f841e62ddd6470554a). It allows us to wire up these stage-dependent values in code instead of resorting to CFN parameters, which have a number of disadvantages.

These values are copy & pasted from CFN parameters. I've already noticed one mistake (fixed here https://github.com/guardian/mobile-save-for-later/pull/66/files#diff-065b0f04fd0514a9237bd47b77be3af1cce559fccda2e8d78ac36326fa4ed85c). Please let me know if you spot any more!

Reply:
Ah, good spot @waisingyiu, I should have mentioned the problem that I found in this area.

Even though the CFN parameter value is the same for `CODE` and `PROD`, the value has actually been overridden (presumably manually, via the console). You can see this using [drift detection](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-stack-drift.html):
 
<img width="1055" alt="image" src="https://user-images.githubusercontent.com/19384074/171122101-20812a7b-1cfd-4022-bd82-118907d2a7fb.png">

The `CODE` lambda doesn't actually work correctly if you try to use `https://id.guardianapis.com/` (when you call it via a debug version of the app).

Reply:
Thank you Jacob for your explanation.  And I learnt a new thing here - drift detection.